django/db/models/functions/__init__.py

```diff
@@ -34,7 +35,7 @@
     'Sin', 'Sqrt', 'Tan',
     # text
     'Chr', 'Concat', 'ConcatPair', 'Left', 'Length', 'Lower', 'LPad', 'LTrim',
-    'Ord', 'Repeat', 'Replace', 'Reverse', 'Right', 'RPad', 'RTrim',
+    'MD5', 'Ord', 'Repeat', 'Replace', 'Reverse', 'Right', 'RPad', 'RTrim',
```

Comment:
Please order this in the same way as the import above.

Reply:
Updated :+1: